projects/eval-adxl355-pmdz/src/common/common_data.c

```diff
@@ -46,44 +46,48 @@
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
 #ifdef DUMMY_EXAMPLE
-struct no_os_uart_init_param uip = {
+struct no_os_uart_init_param adxl355_uart_ip = {
 	.device_id = UART_DEVICE_ID,
-	.baud_rate = 115200,
```

Comment:
is this change related to the commit description? looks like this commit does more than adapting to the new trigger structure.

Reply:
no, this change should go to the code refactoring part. I will update this. thanks!